test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -75,6 +75,15 @@ public EndToEndTests(ServerFixture serverFixture, ITestOutputHelper output) : ba
             }
         }
 
+        [Fact]
+        public async Task CanStartConnectionUsingDefaultTransport()
+        {
+            var url = _serverFixture.BaseUrl + "/echo";
+            var connection = new HttpConnection(new Uri(url));
```

Comment:
Did a little digging and server side we are sending the close frame in response to `CloseAsync` from the client, but the client never finishes the `CloseAsync` call, we filed an issue to change it to `CloseOutputAsync` and this test will probably continue being flaky until we make that change.